database/web3/currency.go

```diff
@@ -4,19 +4,20 @@ import (
 	"math/big"
 	"fmt"
 	"github.com/ethereum/go-ethereum/params"
+	"github.com/ethereum/go-ethereum/common/math"
 )
 
-func PhotonToWei(value string) (*big.Int, error) {
```

Comment:
[MINOR] Why did you remove this method? Does it not look useful? Having us to calculate amounts in Wei it is adding alot of overhead, I would suggest to restore this method unless there is a better alternative to convert PHT to WEI

Reply:
because it was not very useful and it worked only with integers. I replaced it with ParseWei that is more flexible. At the end if Ethereum doesn't handle floats anywhere in the code maybe we shouldn't go that extra mile needed. We can add additional method like PhotonToWEi in the future but only if it will work well with all types, string, integers and floats. For now ParseWei() will do the trick